packages/story-editor/src/utils/storyPageToDataUrl.ts

```diff
@@ -18,6 +18,8 @@
  * External dependencies
  */
 import { PAGE_RATIO } from '@googleforcreators/units';
+import type { Page } from '@googleforcreators/elements';
+import type { Options } from 'html-to-image/lib/types';
```

Comment:
```suggestion
import type { Options } from 'html-to-image';
```

This doesn't work?

Reply:
I'm seeing this error

<img width="754" alt="Screen Shot 2022-12-01 at 5 33 48 AM" src="https://user-images.githubusercontent.com/62242/205030599-5e409b0f-682b-4331-a24e-8d57088f2703.png">